src/occurrences.Rmd

```diff
@@ -20,9 +21,15 @@ library(trias)
 
 ```{r}
 input_checklist <- "../data/input/eu_concern_species.tsv"
-eu_concern_species <- read.table(file = input_checklist, header = TRUE, sep = "\t")
+eu_concern_species <- 
+  read_delim(file = input_checklist, 
+             "\t", escape_double = FALSE, 
+             col_types = cols(
+               entry_into_force = col_date(
+                 format = "%Y-%m-%d")), 
+             trim_ws = TRUE)
 taxonKeys <- eu_concern_species["backbone_taxonKey"]
-countries <- list(country = c("BE"))
+countries <- list(country = c("LU"))
```

Comment:
I'm just wondering, why always the `list` -> `unlist` combination?

Reply:
You are right. Solved in 1bd0b001252458058c157c77c55d407fd7217ca8